README.md

```diff
@@ -311,6 +311,7 @@ _CalDAV or CardDAV clients._
 
 _Simple deployment of a mail server, e.g. for inexperienced or impatient admins._
 
+- [wildduck](https://wildduck.email/) - Opinionated email server ([Source Code](https://github.com/nodemailer/wildduck)) `EUPL-1.2` `Node.js`
```

Comment:
Thank you for contributing.
Please elaborate more on the description.  The description must all end with full stop.
Please add EUPL-1.2 to the list of licenses.

Reply:
Added, please see.